4_Zadania_dodatkowe/6.InfiniteSlider/css/style.css

```diff
@@ -1,10 +1,18 @@
-.slider {
-  height: 250px;
+.slider{
+  height: 200px; /* zmiana z 250 na 200 bo przy 250 widac czesc 2 obrazka */
+  width: 400px;
+  overflow: hidden;
 }
 
-.slider ul {
+.slider ul{
+  position: relative;
   list-style: none;
+  width: 3228px;
+  margin: 0;
+  padding: 0;
+  right: 404px;
```

Comment:
to powoduje, że przy ładowaniu strony na ułamek sekundy, zanim wykona się kod jquery, widać drugie zdjęcie, to z nosorożcami
to taki niemiły efekt, którego da się uniknąć